snaps_boot/ansible_p/commission/hardware/playbooks/setIsolCpus.yaml

```diff
@@ -45,6 +45,8 @@
      when: (not(isolcpus == "") or not(hugepages == ""))
 
    - name: reboot the system
-     command: reboot
+     shell: sleep 2 && /sbin/shutdown -r now "Ansible system package upgraded"
+     async: 1
+     poll: 0
      ignore_errors: true
      when: (not(isolcpus == "") or not(hugepages == ""))
```

Comment:
Should add in a wait for host to come back up something like:
- name: Wait for server to restart
  local_action:
    module: wait_for
      host={{ inventory_hostname }}
      port=22
      delay=10
    become: false

Reply:
Let's leave this for another patch.  This needs to be done for -b, -s, -i and the clean commands.